src/ucp/core/ucp_listener.h

```diff
@@ -34,6 +34,7 @@ typedef struct ucp_listener {
                                                  creates a handle to
                                                  connection request to the
                                                  remote endpoint */
+    int32_t                        conn_reqs; /* count connection requests */
```

Comment:
unsigned

Reply:
@yosefe 
The unprocessed connection requests may not be more than ```2^31``` (qp number is 24 bits, socket fd number should be limited by system configuration and memory resources).
Could it keep using ```int32_t``` here to give obvious error info if there's something wrong in the code implmentation when the ```conn_reqs``` is lesss than 0?
If there's no need to do so, I'll change it to be ```unsigned```.

Reply:
then, use int

Reply:
Done. Use ```int``` type here.